source/topologies.txt

```diff
@@ -0,0 +1,36 @@
+============================
+Supported Cluster Topologies
+============================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+:ref:`c2c-mongosync` supports two cluster topologies: 
+
+- :ref:`Replica sets <replication>`
+
+- :ref:`Sharded clusters <sharded-cluster>`
+
+To sync a replica set to a replica set, see the :ref:`Quick Start Guide
+<c2c-quickstart>`.
+
+To sync a replica set to a sharded cluster, see :ref:`c2c-rs-to-sharded`.
+
+To sync a sharded cluster to a sharded cluster, see :ref:`c2c-sharded-clusters`.
+
+.. note::
+   
+   - If you are syncing sharded clusters, they are not required to have the same 
+     number of shards.
```

Comment:
Just wondering if[ you saw the ruler setup in the wiki,](https://wiki.corp.mongodb.com/display/DE/Install+and+Configure+Documentation+Tools)  (search Preferences > Settings) we usually assign a length of 80 characters per line and to delimit our line breaks. This helps with lines being broken up on GIthub PR reviews.

Reply:
Weird, I have the ruler lines set up according to that wiki, and this line shouldn't be going past 80 characters because I manually (I don't have text wrapping on because it messes with my indents). I may slack you about this because I am not sure what I am doing wrong. Thanks for the heads up!